### DISCUSSION TITLE
Shrink text in pptx presentation

### DISCUSSION THREAD

**Discussion by aldotapia** [quarto-dev/quarto-cli#1502] [Q&A]
Hi, 

I'm rendering a pptx presentation and the text isn't compressing in text blocks. In my template I have `shrink text on overflow` activated. Also, I'm using a CSS style for reducing the font size and it isn't working (`font-size: 0.5em;`)

I'm using RStudio 2022.02.3 for rendering the presentation.

Any hint about this?
<img width="1048" alt="Captura de Pantalla 2022-07-22 a la(s) 17 54 55" src="https://user-images.githubusercontent.com/23053094/180574132-1ddd6872-df66-4268-bd4c-f0cd017be344.png">

**Comment by danielinteractive** [comment]
Thanks @aldotapia , I would have the same question - even just setting the font size manually on a specific slide in Quarto for pptx would be helpful as a first workaround

**Comment by cderv** [comment]
Powerpoint styling is possible in the limitation of what is possible in Pandoc. We mentions the template feature in https://quarto.org/docs/presentations/powerpoint.html#powerpoint-templates which is directly based on Pandoc supports: https://pandoc.org/MANUAL.html#option--reference-doc

Though it remains quite limited because [Custom Styles](https://pandoc.org/MANUAL.html#custom-styles) is not supported by Pandoc

> Also, I'm using a CSS style for reducing the font size and it isn't working (font-size: 0.5em;)

CSS won't work with PPTX. And `font-size` is not among the option available for PPTX format (https://quarto.org/docs/reference/formats/presentations/pptx.html)

So currently Powerpoint features is limited by Pandoc capabilities.

**Reply by danielinteractive** [reply]
Thanks @cderv that is good to know! Is there a Pandoc repository where one could try to contribute or make a feature request for a font size option?

**Reply by mcanouil** [reply]
The GitHub repository: https://github.com/jgm/pandoc
Although as for docx, pptx follows the same principle, which is to define a reference document with all the styling you want.

**Reply by cderv** [reply]
You get the link of the repo above by @mcanouil. Though, seeing his answer, I may have been not clear enough before. 

PPTX is customizable by a reference doc like DOCX. However, custom styling using Divs and Spans is only working for DOCX. so you can customize some default layout, but you won't be able to apply custom styling on individual content. 
In docx you could.
